fuzzers/others/libafl-fuzz/Makefile.toml

```diff
@@ -222,6 +232,43 @@ test -n "$( ls ./test/output-unicorn-cmpcov/fuzzer_main/queue/id:000002* 2>/dev/
 '''
 dependencies = ["build_libafl_fuzz", "build_afl", "build_unicorn_mode"]
 
+[tasks.test_nyx_mode]
+script_runner = "@shell"
+script = '''
+export AFL_PATH=${AFL_DIR}
+export AFL_CORES=1
+export AFL_STATS_INTERVAL=1
+export AFL_DEBUG=1 
+export LIBAFL_DEBUG_OUTPUT=1
+
+AFL_PATH=${AFL_DIR} ${AFL_CC_PATH} ./test/test-instr.c -o ./test/out-instr
+
+rm -rf ./test/nyx-test
+
+python3 ${AFL_DIR}/nyx_mode/packer/packer/nyx_packer.py \
```

Comment:
to integrate this in ci we would need a dedicated machine.
for now I'll just run the test on my machine and see if it works when @R9295 replies

Reply:
Thanks @tokatoka  & @rmalmain . I've got access to an Intel machine now. I'll change libafl-fuzz's nyx mode to use libafl_nyx instead of AFL++'s nyx_mode cause libnyx is Rust anyways so IMO it makes more sense